modules/olm-installing-from-operatorhub-using-web-console.adoc

```diff
@@ -77,18 +77,25 @@ endif::[]
 ifdef::olm-user[]
 .. Choose a specific, single namespace in which to install the Operator. The Operator will only watch and be made available for use in this single namespace.
 endif::[]
-.. If the cluster is in AWS STS mode, enter the Amazon Resource Name (ARN) of the AWS IAM role of your service account in the *role ARN* field.
+
+.. For clusters on cloud providers with token authentication enabled:
++
+--
+* If the cluster is in AWS STS mode, enter the Amazon Resource Name (ARN) of the AWS IAM role of your service account in the *role ARN* field.
 +
 image::oadp-install-operator-role-arn.png[Entering the ARN]
 To create the role's ARN, follow the procedure described in link:https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/tutorials/cloud-experts-deploy-api-data-protection#prepare-aws-account_cloud-experts-deploy-api-data-protection[Preparing AWS account].
 
+* If the cluster is in {azure-id} mode, add the client ID, tenant ID, and subscription ID in the appropriate field.
```

Comment:
When I originally added the AWS STS bullet before this Azure one, I did not include a screenshot, tho it looks like a different PR has since added one in. I don't really want to include a screenshot here for Azure, especially when it would be 95% the same as the one that is now up in the AWS bullet.

Might consider chatting with the previous PR author about whether the AWS screenshot is really needed.

Reply:
I'd be in favor of taking it out. I don't think we normally do screenshots because the GUI changes so often so CI gets to be too tough to keep up with.